### PR TITLE
added db persistance

### PR DESCRIPTION
**Pull request by tgoyal63**

### CODE REVIEW

File: src/commands/text/moderation/ban.command.ts
Comment (Teoriya) [inline src/commands/text/moderation/ban.command.ts:1]: @tgoyal63

File: src/routes/config.routes.ts
Comment (Teoriya) [inline src/routes/config.routes.ts:1]: idk

Comment (tgoyal63) [reply src/routes/config.routes.ts:1]: Figure it out

File: src/commands/text/moderation/ban.command.ts
Comment (tgoyal63) [reply src/commands/text/moderation/ban.command.ts:1]: Sahi hai

**Pull request merged**